Gems/AWSCore/Code/Include/Private/Editor/UI/AWSCoreResourceMappingToolAction.h

```diff
@@ -10,20 +10,25 @@
 #include <AzCore/std/string/string.h>
 
 #include <QAction>
+#include <QObject>
 
 namespace AWSCore
 {
     class AWSCoreResourceMappingToolAction
         : public QAction
     {
     public:
+        static constexpr const char AWSCoreResourceMappingToolActionName[] = "AWSCoreResourceMappingToolAction";
         static constexpr const char ResourceMappingToolDirectoryPath[] = "Gems/AWSCore/Code/Tools/ResourceMappingTool";
+        static constexpr const char ResourceMappingToolLogDirectoryPath[] = "user/log/";
```

Comment:
q: don't we have the special @user signifier for this type of path?

Reply:
We do, it gets resolved via the FileIOBase instance